lib/services/ios-log-filter.ts

```diff
@@ -41,14 +41,14 @@ export class IOSLogFilter extends iOSLogFilterBase.IOSLogFilter implements Mobil
 				// This code removes unnecessary information from log messages. The output looks like:
 				// CONSOLE LOG file:///location:row:column: <actual message goes here>
 				if (pid) {
-					const searchString = "[" + pid + "]: ";
-					const pidIndex = line.indexOf(searchString);
-					if (pidIndex > 0) {
-						line = line.substring(pidIndex + searchString.length, line.length);
+					if (line.indexOf(`[${pid}]: `) !== -1) {
+						const pidRegex = new RegExp(`.*[\\[]{1}${pid}[\\]]{1}(?::\\s(:?\\(NativeScript\\)\\s)*)`);
```

Comment:
The `RegExp` can be shortened to:
`` const pidRegex = new RegExp(`.*[\\[]${pid}[\\]]:\\s(:?\\(NativeScript\\)\\s)*`); ``

Reply:
How does this look as a final version:
```javascript
const pidRegex = new RegExp(`^.*[\\[]${pid}[\\]]:\\s(?:\\(NativeScript\\)\\s)?\`);
```